### PR TITLE
Add Styles to companies scaffold

### PR DESCRIPTION
**Pull request by adrianMRJ**
# Description
## What did you implemented/Why did you implemented this:
 - We've added the styles to our scaffold of companies 
 -  These changes were made to complement the part of user view

# Screenshots

Index-Desktop

Edit-Desktop

New-Desktop

Show-Desktop

### CODE REVIEW

File: app/views/companies/_form.html.erb [hunk 1]
```diff
@@ -11,18 +11,26 @@
     </div>
   <% end %>
 
-  <div class="field">
-    <%= form.label :name %>
-    <%= form.text_field :name %>
+  <div class="field text-lg md:text-2xl">
+    <p class="my-2"><%= form.label :name %><br /></p>
```
Comment (YilKanda) [inline app/views/companies/_form.html.erb:15]: Why do you use a line break (br) inside a paragraph (p) tag?

File: app/views/companies/_form.html.erb [hunk 2]
```diff
@@ -11,18 +11,26 @@
...
+      class: "h-10 md:text-xl md:h-14 w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline shadow appearance-none border text-center rounded",
+      placeholder: "Name"
+    %>
   </div>
 
-  <div class="field">
-    <%= form.label :address %>
-    <%= form.text_field :address %>
+  <div class="field text-lg md:text-2xl">
+    <p class="my-2"><%= form.label :address %><br /></p>
```
Comment (YilKanda) [inline app/views/companies/_form.html.erb:23]: Why do you use a line break (br) inside a paragraph (p) tag?

File: app/views/companies/index.html.erb [hunk 3]
```diff
@@ -1,29 +1,33 @@
-<p id="notice"><%= notice %></p>
 
-<h1>Companies</h1>
+  <div class="flex justify-center">
+    <div class="appearance-none rounded w-full py-4 md:py-16 px-9 md:px-48 text-gray-700 leading-tight focus:outline-none focus:shadow-outline flex flex-col items-center">
+      <p id="notice"><%= notice %></p>
```
Comment (YilKanda) [inline app/views/companies/index.html.erb:4]: To the notifications, will not you put styles?

File: app/views/companies/index.html.erb [hunk 4]
```diff
@@ -1,29 +1,33 @@
...
+      <% end %>
 
-<br>
+      <br>
 
-<%= link_to 'New Company', new_company_path %>
+      <%= link_to 'New Company', new_company_path, class:"  mt-2 text-center h-10 pt-2 md:h-14 flex-shrink-0 bg-gray-900 hover:bg-gray-600 border-teal-500 hover:border-teal-700 text-sm md:text-xl w-full md:w-1/2 text-white py-1 px-2 rounded" %>
+    </div>
+  </div>
+
```
Comment (YilKanda) [inline app/views/companies/index.html.erb:31]: Here's an extra line.

File: app/views/companies/edit.html.erb [hunk 5]
```diff
@@ -1,6 +1,11 @@
-<h1>Editing Company</h1>
+<div class="flex justify-center">
+  <div class="flex flex-col items-center shadow appearance-none border rounded py-4 md:py-16 px-9 md:px-48 text-gray-700 leading-tight focus:outline-none focus:shadow-outline">
+    <div class="font-sans font-semibold text-center m-2 text-2xl md:text-4xl md:pb-8">
+      <h2>EDITING COMPANY</h2>
+    </div>
 
-<%= render 'form', company: @company %>
+    <%= render 'form', company: @company %>
```
Comment (JuanequeX) [inline app/views/companies/edit.html.erb:7]: Delete this blank lines

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Pull request merged**